From ed0a47c22843fcefa4c3ed8fabf5a989e6c1c538 Mon Sep 17 00:00:00 2001
From: bartus <szczepaniak.bartek+github@gmail.com>
Date: Wed, 8 Jan 2025 16:32:34 +0100
Subject: [PATCH] Pop cxx standard for poppler:25

---
 OpenBoard.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OpenBoard.pro b/OpenBoard.pro
index 17fb4e75a..ba0d67601 100644
--- a/OpenBoard.pro
+++ b/OpenBoard.pro
@@ -1,7 +1,7 @@
 TARGET = "OpenBoard"
 TEMPLATE = app
 
-CONFIG += c++17
+CONFIG += c++20
 CONFIG -= flat
 CONFIG += debug_and_release \
           no_include_pwd
-- 
2.47.1

